addons/crm_iap_lead/models/crm_iap_lead_industry.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CrmIapLeadIndustry(models.Model):
```

Comment:
Seems you forgot to change init.py in commit about move (is done in second commit)